yt/go/yt/interface.go

```diff
@@ -677,7 +677,7 @@ type OperationStatus struct {
 	FullSpec          yson.RawValue              `yson:"full_spec"`
 	StartTime         yson.Time                  `yson:"start_time"`
 	FinishTime        yson.Time                  `yson:"finish_time"`
-	Suspend           bool                       `yson:"suspend"`
+	Suspend           bool                       `yson:"suspended"`
```

Comment:
Let's rename the field name too. I doubt that there is much code that would break, taking into account that the field was never working.

Reply:
Let's keep both for some time maybe? Even though it was not working, some code could rely on its existence

Reply:
And it seems it worked for rpc client

Reply:
okay